000-index.md

```diff
@@ -5,7 +5,7 @@ weight: 001
 
 # CNAB Specifications
 
-1. [Cloud Native Application Bundle Core 1.0.0 (CNAB1)](100-CNAB.md)
+1. [Cloud Native Application Bundle Core 1.1.0 (CNAB1)](100-CNAB.md)
```

Comment:
Q: Does `CNAB1` need to change?

Reply:
I would prefer if we have CNAB<major> mean the latest CNAB release of the <major> version (i.e.: In this case, CNAB1 means CNAB 1.1.0) as I think that usually this is what people mean when they say version <major>. If one wants to be more specific, they can do so by specifying the minor and patch versions as well.

If we choose to do this, we may need to clarify our [process versioning](https://github.com/cnabio/cnab-spec/blob/master/901-process.md#versioning).